0-original-walkthrough.md

```diff
@@ -758,48 +758,46 @@ e.g. `PROJ_P1_Li`.
 
 <a name="7"/><a href="#">top</a>
 
-# 7. Dilutions, Pools, and Orders
+# 7. Library Aliquots, Pools, and Orders
 
 A library cannot be directly loaded into a _lane_ in a _sequencing container_
-(flowcell/SMRTcell) in MISO. A dilution must be made and then many dilutions
+(flowcell/SMRTcell) in MISO. A library aliquot must be made and then many aliquots
 (or just one) can be mixed into a _pool_ for sequencing.
 
 _Orders_ are requests for sequencing pools a certain number of times. They are
 used to keep track of sequencing progress for project management and book-keeping. 
 
-## 7.1 Bulk creating Library Dilutions
-Dilutions can be made in bulk from libraries.
+## 7.1 Bulk creating Library Aliquots
+Library Aliquots can be made in bulk from libraries.
 
-<img src="pics/flow-dilution.svg"/>
+<img src="pics/flow-library-aliquot.svg"/>
 
-In this exercise, we will create 4 library dilutions from the libraries we
+In this exercise, we will create 4 library aliquots from the libraries we
 made previously.
 
 1. On the _Libraries_ page, check all the libraries just created.
   - `PROJ_0001_Br_P_PE_300_EX`
   - `PROJ_0001_Ly_R_PE_300_EX`
   - `PROJ_0002_Br_P_PE_300_EX`
   - `PROJ_0002_Ly_R_PE_300_EX`
-1. From the _Bulk actions_ drop down, select _Make dilutions from selected_ and click _Go_.
-1. Enter the concentrations of the dilutions (use any number you wish).
+1. Click the _Make aliquots_ button at the top of the table. In the dialog that appears,
+   click _Create_.
+1. Enter the concentrations of the aliquots (use any number you wish).
 1. Click _Save_.
 
-Dilutions are a bit ephemeral in MISO: there is no list for all the dilutions.
-To see the dilutions for a library, view the library page. At present,
-Dilutions cannot be saved in Boxes.
-
 ## 7.2 Pools
 Pools are the last step for libraries before sequencing and represent the
 entity that is loaded onto the flowcell lane or SMRTcell. A "pool" can have one or more
-library dilutions in it. They are equivalent to "worksets" in Geospiza, but are
+library aliquots in it. They are equivalent to "worksets" in Geospiza, but are
```

Comment:
This line about pools being `equivalent to "worksets" in Geospiza` may have outlived its usefulness... especially as MISO now has worksets.

Reply:
Didn't see that. Will happily remove, along with the other couple of references to Geospiza!